### ISSUE TITLE
2.7.0-rc1 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2487]
# Minor Release

## 2.7.0-rc1
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2.  Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

### New changes (not included in 2.6.x)
Dropping support for PHP 7.4 and 8.0. 
New minimum PHP version is 8.1

#### New modules
- [Flag](https://www.drupal.org/project/flag)
- [Asset Injector](https://www.drupal.org/project/asset_injector)
- [Media Entity File Replace](https://www.drupal.org/project/media_entity_file_replace)

#### Permissions changes
- Added permissions for Content Administrators and Content Editors for managing shortcuts
- Added permissions for Content Administrators to manage Flags
- Added permissions for Content Administrators to manage their own URL Aliases

#### Deprecated modules slated for removal
- [Entity Browser](https://www.drupal.org/project/entity_browser)

#### New feature
- #2406
- #2480
- #2358
- #2481
- #2461
- #2484
- #2163
- #2262
- #2470
- #2467
- #2291
- #2432
- #2390
- #2308
- #2376
- #2396
- #2393
- #2316
- #2369
- #2311

#### Bug fix
- #2411
- #2364

#### Breaking or visual change to existing behavior
- #2474
- #2381

#### Drupal core major level update
- #2354 

#### Drupal contrib project updates
- #2468
- #2410
- #2428
- #2445
- #2399
- #2372
- #2388
- #2248
- #2269
- #2427
- #2290
- #2193
- #2279
- #2280
- #2450 
- #2337 
- #2452
- #1999
- #2410
- #2248
- #2336
- #2388
- #2372
- #2325
- #2333
- #2397

#### Other
- #2244
- #2246
- #2492
- #2493

#### Bug fix
- #2373
- #2379
- #2320
- #2318
- #2281
- #2278
- #2283
- #2292
- #2260
- #2254
- #2250
- #2252
- #2228
- #2232
- #2469

#### Experimental module feature
- #2223
- #2231
- #2236
- #2274
- #2430
- #2241

#### Drupal core - Patch level release
- #2391
- #2349
- #2306
- #2297
- #2266

#### Drupal contrib - Patch level release
- #2465
- #2459
- #2444
- #2287
- #2238
- #2230
- #2235

#### Other
- #2285
- #2284
- #2218
- #2425
- #2438
- #2343
- #2271
- #2270
- #2344

### PRs to finish/approve/merge

- [x] #2450 
- [x] (Joe) #2337 
	- [x]  Re-export config entity generated by DB update
- [x] #2452  
- [x] #1999
- [x] (Joe) #2354
	- [ ] Re-export all config (?)

### Other tasks

- [ ] (Chris) Generate release notes snippet
```
- Closes #2081: Upgrade Drupal core to 10.0.9 (#2354)
- Update drupal/google_tag requirement from 1.6.0 to 2.0.2 (#2337)
- Closes #2486: Remove tooling for Drupal console 😢. (#2489)
- Closes #2442: Update drupal/bootstrap_barrio from 4.30 to 5.1.10. (#2452)
- Update drupal/better_exposed_filters requirement from 5.0 to 6.0.3 (#1999)
- #2213: Update drupal/migrate_tools requirement from 5.2.0 to 6.0.1 (#2450)
- Closes #2383 Upgrade tag page view. (#2406)
- Closes #2366 Allow linking images that are embedded in content. (#2480)
- Fixes #2247 Publication BibTeX Import (#2358)
- Closes #2321: Update drupal/menu_block requirement from 1.7.0 to 1.10.0 (#2325)
- Closes #2380 Adding permissions for using shortcuts. (#2481)
- Fixes #2440 Publication Translations (#2461)
- Closes #2383 Enable az_digital_asset_library by default (#2484)
- Closes #2473: Update display of links and CV fields in default Person display (#2474)
- Closes #2162, Closes #2477 Update drupal/intelligencebank from 3.0.0 to 4.0.0. (#2163)
- Fixes #2255, Fixes #2299 API Endpoint for News Export (#2262)
- Fixes #2359 Add Curated Publication Listings. (#2470)
- Fixes #2439 Add Publication Image to Full View Display and Card View Display (#2467)
- Fixes #2362 Publication Views exposed filters overlap (#2469)
- Update drupal/entity_embed requirement from 1.3.0 to 1.4.0 (#2468)
- Closes #2464 Change source of bootstrap-datepicker from bower-asset to npm-asset (#2465)
- Closes #2458 Update az_carousel required modules: slick, slick_views, blazy. (#2459)
- Closes #2289: Add basic linkchecker extractor for az_card, and az_accordion field types. (#2291)
- Update bower-asset/bootstrap-datepicker requirement from 1.9.0 to 1.10.0 (#2410)
- Close #2424 Adding badge display for all default taxonomy vocabularies. (#2425)
- Closes #2431 Allow content administrators to manage their own url aliases. (#2432)
- Closes #2382 - Fix PHP 8 code deprecation in custom smart_date migrate plugin. (#2444)
- Closes #1396: Added 3-col view display for AZ Person (#2381)
- Update drupal/ckeditor_bs_grid requirement from 2.0.8 to 2.0.9 (#2428)
- #2213: Update drupal/migrate_queue_importer requirement from 2.0 to 2.1.0 (#2445)
- #2403: Refactor AllowDrupalLogin test trait to not override parent::setup(). (#2411)
- Closes #2417: Fix double-encoded publication file URLs. (#2430)
- Update citation-style-language/styles requirement from 0.0.237 to 0.0.249 (#2427)
- Closes #2437 - Fix Slack channel and meeting information in CONTRIBUTING.md (#2438)
- Update citation-style-language/styles requirement from 0.0.199 to 0.0.237 (#2399)
- Closes #2361 - Add multilingual support (#2390)
- Closes #2091, #1605, #1772 Add an optional thumbnail image field to AZ News, AZ Event (#2308)
- Update drupal/core-recommended requirement from 9.5.8 to 9.5.9 (#2391)
- Closes #2375 Adding Flag module to Quickstart with Basic Permissions (#2376)
- Update drupal/chosen requirement from 3.0.5 to 4.0.0 (#2333)
- Update drupal/asset_injector requirement from 2.16.0 to 2.17.0 (#2397)
- Closes #2395 - Enable Splitscreen by default (#2396)
- Closes #2363: Use ARM64-compatible WebDriver image for Lando. (#2364)
- Update drupal/draggableviews requirement from 2.1.1 to 2.1.3 (#2372)
- Closes #2106 Allow <code> and/or <pre> HTML elements in az_standard text format (#2393)
- Update drupal/link_class requirement from 2.0.0 to 2.1.0 (#2388)
- Fixes #2313 Update Citation Style Configuration (#2316)
- Closes #2371: Transform IDs in both source and destination paths for migrated redirects. (#2373)
- Closes #2368 Add option to move Sidebar First region below page content on mobile (#2369)
- Fixes #2378 Academicpuma/locales abandoned (#2379)
- Update drupal/linkit requirement from 6.0.0-beta4 to 6.0.0-rc1 (#2336)
- Closes #2348: Update Drupal core to 9.5.8 for SA-CORE-2023-005 (#2349)
- Closes #2342: Change links to chili on Event displays (#2344)
- Closes #2341: Enable use of description on Person's CV field (#2343)
- Closes #2265 Add manual migration lookup process plugin for migrate. (#2271)
- Update drupal/core-recommended requirement from 9.5.5 to 9.5.7 (#2306)
- Documentation: Update various pre-migration suggestions to include checking pathauto settings (#2285)
- Closes #1528: Allow users to replace files, also rename files through re-upload (#2311)
- Closes #2315: Downgrade menu_block to 8.x-1.7 until we have a better solution for empty sidebars getting rendered. (#2320)
- Fixes #2312 Changes to existing news, event and person content do not update without clearing cache. (#2318)
- Closes #2264 Migrate path aliases for nodes if they exist, otherwise turn on Pathauto. (#2281)
- Update drupal/quick_node_clone requirement from 1.15 to 1.16.0 (#2290)
- Update drupal/search_exclude requirement from 2.0.0-beta2 to 3.0.0-beta1 (#2193)
- Update drupal/inline_entity_form requirement from 1.0-rc14 to 1.0.0-rc15 (#2279)
- Closes #2294: Update Drupal core to 9.5.5 (moderately critical security update). (#2297)
- Update drupal/linkit requirement from 6.0.0-beta3 to 6.0.0-beta4 (#2280)
- Fixes #2277 Text Area, and Single Card paragraphs behavior settings migrating incorrectly. (#2278)
- Fixes #2282 News, Event, Person external links do not migrate. (#2283)
- Fixes #2257, Fixes #2258 Publication Person Block Title, Add Publication Taxonomy (#2270)
- Closes #2286 Update allowed Smart Date 3.7.x version from 3.7.0 to 3.7.2 (#2287)
- Documentation: Add pre-migration task to menu links migration docs. (#2284)
- Closes #2288 Fix broken az_barrio footer logo field validation (#2292)
- Closes #2268: Add drupal/asset_injector as optional module. (#2269)
- Closes #2273: Fix az_events_trellis coding standards issues. (#2274)
- Update drupal/core-recommended requirement from 9.5.3 to 9.5.4 (#2266)
- Closes #2259 use null coallesce operator when creating strings to print from deeply nested array values (ParagraphsExtraInfoFieldMerge). (#2260)
- Closes #2253 - Add az_flexible_page to smart_title settings (#2254)
- Closes #2249: Fix migration config for link field on news and events that is generating PHP warnings. (#2250)
- Closes #2251 Remove block_class config for `az_barrio_search` block and add `az-search-block` class via preprocess hook. (#2252)
- Update drupal/media_migration requirement from 1.0.0-alpha14 to 1.0.0-alpha16 (#2248)
- Closes #2119 Add categories to events and pages in demo content (#2244)
- Closes #2245 Unused imports in flexible page settings form (#2246)
- Closes #2239: Remove unnecesssary timezone processing from Trellis Events importer. (#2241)
- #2234: Allow smart_date 3.6.1 to be used since 3.7.0 requires PHP 8.0+. (#2238)
- Closes #2227 re-add linked image in az_person az_row view mode. (#2228)
- Fixes #2233 Make Publication citation style not dependent on View Mode (#2236)
- Fixes #2214 Add a View to Publications to have all searches in one View (#2231)
- Update drupal/workbench_access requirement from 2.0.0 to 2.0.1 (#2230)
- Closes #2234: Update drupal/smart_date from 3.6.1 to 3.7.0. (#2235)
- Closes #2229: Ensure block_class module is installed. (#2232)
- Closes #2222: Require PHP 8.0+ for experimental Trellis Events integration module. (#2223)
- Update composer branch alias in preparation for 2.6.0 pre-release. (#2218)
```

**Comment by camikazegreen** [comment]
https://github.com/az-digital/az_quickstart/pull/2344 was already included in the 2.6.3 patch release.

**Comment by joeparsons** [comment]
Release notes snippet:
```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

## New changes (not included in 2.6.x)
### New modules
- [Flag](https://www.drupal.org/project/flag)
- [Asset Injector](https://www.drupal.org/project/asset_injector)

### Deprecated modules slated for removal
- [Entity Browser](https://www.drupal.org/project/entity_browser)

### New feature
- Closes #2383 Upgrade tag page view. #2406
- Closes #2366 Allow linking images that are embedded in content. #2480
- Fixes #2247 Publication BibTeX Import #2358
- Closes #2380 Adding permissions for using shortcuts. #2481
- Fixes #2440 Publication Translations #2461
- Closes #2383 Enable az_digital_asset_library by default #2484
- Closes #2162, Closes #2477 Update drupal/intelligencebank from 3.0.0 to 4.0.0. #2163
- Fixes #2255, Fixes #2299 API Endpoint for News Export #2262
- Fixes #2359 Add Curated Publication Listings. #2470
- Fixes #2439 Add Publication Image to Full View Display and Card View Display #2467
- Closes #2289: Add basic linkchecker extractor for az_card, and az_accordion field types. #2291
- Closes #2431 Allow content administrators to manage their own url aliases. #2432
- Closes #2361 - Add multilingual support #2390
- Closes #2091, #1605, #1772 Add an optional thumbnail image field to AZ News, AZ Event #2308
- Closes #2375 Adding Flag module to Quickstart with Basic Permissions #2376
- Closes #2395 - Enable Splitscreen by default #2396
- Closes #2106 Allow <code> and/or <pre> HTML elements in az_standard text format #2393
- Fixes #2313 Update Citation Style Configuration #2316
- Closes #2368 Add option to move Sidebar First region below page content on mobile #2369
- Closes #1528: Allow users to replace files, also rename files through re-upload #2311

### Bug fix
- #2403: Refactor AllowDrupalLogin test trait to not override parent::setup(). #2411
- Closes #2363: Use arm64-compatible webdriver image for lando. #2364

### Breaking or visual change to existing behavior
- Closes #2473: Update display of links and CV fields in default Person display #2474
- Closes #1396: Added 3-col view display for AZ Person #2381


### Drupal core major level update
- Closes #2081: Upgrade Drupal core to 10.0.9 #2354

### Drupal contrib project updates
- Update drupal/entity_embed requirement from 1.3.0 to 1.4.0 #2468
- Update bower-asset/bootstrap-datepicker requirement from 1.9.0 to 1.10.0 #2410
- Update drupal/ckeditor_bs_grid requirement from 2.0.8 to 2.0.9 #2428
- #2213: Update drupal/migrate_queue_importer requirement from 2.0 to 2.1.0 #2445
- Update citation-style-language/styles requirement from 0.0.199 to 0.0.237 #2399
- Update drupal/draggableviews requirement from 2.1.1 to 2.1.3 #2372
- Update drupal/link_class requirement from 2.0.0 to 2.1.0 #2388
- Update drupal/media_migration requirement from 1.0.0-alpha14 to 1.0.0-alpha16 #2248
- Closes #2268: Add drupal/asset_injector as optional module. #2269
- Update citation-style-language/styles requirement from 0.0.237 to 0.0.249 #2427
- Update drupal/quick_node_clone requirement from 1.15 to 1.16.0 #2290
- Update drupal/search_exclude requirement from 2.0.0-beta2 to 3.0.0-beta1 #2193
- Update drupal/inline_entity_form requirement from 1.0-rc14 to 1.0.0-rc15 #2279
- Update drupal/linkit requirement from 6.0.0-beta3 to 6.0.0-beta4 #2280
- #2213: Update drupal/migrate_tools requirement from 5.2.0 to 6.0.1 #2450
- Update drupal/google_tag requirement from 1.6.0 to 2.0.2 #2337
- Closes #2442: Update drupal/bootstrap_barrio from 4.30 to 5.1.10. #2452
- Update drupal/better_exposed_filters requirement from 5.0 to 6.0.3 #1999
- Update bower-asset/bootstrap-datepicker requirement from 1.9.0 to 1.10.0 #2410
- Update drupal/media_migration requirement from 1.0.0-alpha14 to 1.0.0-alpha16 #2248
- Update drupal/linkit requirement from 6.0.0-beta4 to 6.0.0-rc1 #2336
- Update drupal/link_class requirement from 2.0.0 to 2.1.0 #2388
- Update drupal/draggableviews requirement from 2.1.1 to 2.1.3 #2372
- Closes #2321: Update drupal/menu_block requirement from 1.7.0 to 1.10.0 #2325
- Update drupal/chosen requirement from 3.0.5 to 4.0.0 #2333
- Update drupal/asset_injector requirement from 2.16.0 to 2.17.0 #2397

### Other
- Closes #2119 Add categories to events and pages in demo content #2244
- Closes #2245 Unused imports in flexible page settings form #2246


## Backported changes (also included in 2.6.x)
### Bug fix
- Closes #2371: Transform IDs in both source and destination paths for migrated redirects. #2373
- Fixes #2378 Academicpuma/locales abandoned #2379
- Closes #2315: Downgrade menu_block to 8.x-1.7 until we have a better solution for empty sidebars getting rendered. #2320
- Fixes #2312 Changes to existing news, event and person content do not update without clearing cache. #2318
- Closes #2264 Migrate path aliases for nodes if they exist, otherwise turn on Pathauto. #2281
- Fixes #2277 Text Area, and Single Card paragraphs behavior settings migrating incorrectly. #2278
- Fixes #2282 News, Event, Person external links do not migrate. #2283
- Closes #2288 Fix broken az_barrio footer logo field validation #2292
- Closes #2259 use null coallesce operator when creating strings to print from deeply nested array values (ParagraphsExtraInfoFieldMerge). #2260
- Closes #2253 - Add az_flexible_page to smart_title settings #2254
- Closes #2249: Fix migration config for link field on news and events that is generating PHP warnings. #2250
- Closes #2251 Remove block_class config for az_barrio_search block and add az-search-block class via preprocess hook. #2252
- Closes #2227 re-add linked image in az_person az_row view mode. #2228
- Closes #2229: Ensure block_class module is installed. #2232
- Fixes #2362 Publication Views exposed filters overlap #2469

### Experimental module feature
- Closes #2222: Require PHP 8.0+ for experimental Trellis Events integration module. #2223
- Fixes #2214 Add a View to Publications to have all searches in one View #2231
- Fixes #2233 Make Publication citation style not dependent on View Mode #2236
- Closes #2273: Fix az_events_trellis coding standards issues. #2274
- Closes #2417: Fix double-encoded publication file URLs. #2430
- Closes #2239: Remove unnecesssary timezone processing from Trellis Events importer. #2241

### Drupal core - Patch level release
- Update drupal/core-recommended requirement from 9.5.8 to 9.5.9 #2391
- Closes #2348: Update Drupal core to 9.5.8 for SA-CORE-2023-005 #2349
- Update drupal/core-recommended requirement from 9.5.5 to 9.5.7 #2306
- Closes #2294: Update Drupal core to 9.5.5 (moderately critical security update). #2297
- Update drupal/core-recommended requirement from 9.5.3 to 9.5.4 #2266

### Drupal contrib - Patch level release
- Closes #2464 Change source of bootstrap-datepicker from bower-asset to npm-asset #2465
- Closes #2458 Update az_carousel required modules: slick, slick_views, blazy. #2459
- Closes #2382 - Fix PHP 8 code deprecation in custom smart_date migrate plugin. #2444
- Closes #2286 Update allowed Smart Date 3.7.x version from 3.7.0 to 3.7.2 #2287
- #2234: Allow smart_date 3.6.1 to be used since 3.7.0 requires PHP 8.0+. #2238
- Update drupal/workbench_access requirement from 2.0.0 to 2.0.1 #2230
- Closes #2234: Update drupal/smart_date from 3.6.1 to 3.7.0. #2235

### Other
- Documentation: Update various pre-migration suggestions to include checking pathauto settings #2285
- Documentation: Add pre-migration task to menu links migration docs. #2284
- Update composer branch alias in preparation for 2.6.0 pre-release. #2218
- Close #2424 Adding badge display for all default taxonomy vocabularies. #2425
- Closes #2437 - Fix Slack channel and meeting information in CONTRIBUTING.md #2438
- Closes #2341: Enable use of description on Person's CV field #2343
- Closes #2265 Add manual migration lookup process plugin for migrate. #2271
- Fixes #2257, Fixes #2258 Publication Person Block Title, Add Publication Taxonomy #2270
- Closes #2342: Change links to chili on Event displays #2344
```

**Comment by trackleft** [comment]
[2.7.0-rc1](https://github.com/az-digital/az_quickstart/releases/tag/2.7.0-rc1)
